src/client/index.js

```diff
@@ -1,14 +1,18 @@
 import { useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { ApolloProvider, ApolloClient, ApolloLink, gql } from '@apollo/client';
+// import { createPersistedQueryLink } from '@apollo/client/link/persisted-queries';
+// import { sha256 } from 'react-native-sha256';
```

Comment:
These lines

Reply:
is `react-native-sha256` added to the pkg json?

Reply:
Yeah, 1.4.7